packages/perseus/src/widgets/expression/expression.tsx

```diff
@@ -158,25 +121,7 @@ export class Expression extends React.Component<Props, ExpressionState> {
     _textareaId = `expression_textarea_${Date.now()}`;
     _isMounted = false;
 
-    //#region Previously a class extension
-    /* Content creators input a list of answers which are matched from top to
-     * bottom. The intent is that they can include spcific solutions which should
-     * be graded as correct or incorrect (or ungraded!) first, then get more
-     * general.
-     *
-     * We iterate through each answer, trying to match it with the user's input
-     * using the following angorithm:
-     * - Try to parse the user's input. If it doesn't parse then return "not
-     *   graded".
-     * - For each answer:
-     *   ~ Try to validate the user's input against the answer. The answer is
-     *     expected to parse.
-     *   ~ If the user's input validates (the validator judges it "correct"), we've
-     *     matched and can stop considering answers.
-     * - If there were no matches or the matching answer is considered "ungraded",
-     *   show the user an error. TODO(joel) - what error?
-     * - Otherwise, pass through the resulting points and message.
-     */
+    // TODO remove this in favor of just using expressionValidator
     static validate(
```

Comment:
I debated about whether or not to remove this. I decided that I would like to keep the public API consistent while splitting the validation logic and then come back to do a `major` refactor on all widgets at once.